gitreceived/gitreceived.go

```diff
@@ -26,8 +26,14 @@ import (
 )
 
 const PrereceiveHookTmpl = `#!/bin/bash
-set -eo pipefail; while read oldrev newrev refname; do
-[[ $refname = "refs/heads/master" ]] && git archive $newrev | {{RECEIVER}} "$RECEIVE_REPO" "$newrev" | sed -$([[ $(uname) == "Darwin" ]] && echo l || echo u) "s/^/"$'\e[1G\e[K'"/"
+set -eo pipefail;
+git-archive-all() {
+	git checkout $1 > /dev/null
+	git submodule update --init --recursive > /dev/null
```

Comment:
Either pass `--quiet` to both these commands or discard stderr too (i.e. `&>/dev/null`)